plugins/@grouparoo/clickhouse/src/lib/connect.ts

```diff
@@ -0,0 +1,57 @@
+import { ConnectPluginAppMethod, SimpleAppOptions } from "@grouparoo/core";
+import {
+  MySQLConnection,
+  QueryResultObject,
+} from "@grouparoo/mysql/dist/lib/connect";
+import { log } from "actionhero";
+import { ClickHouse } from "clickhouse";
+import mysql from "mysql";
+
+class ClickHouseHTTPConnection implements MySQLConnection {
+  private clickhouse: ClickHouse;
+
+  public constructor(appOptions: SimpleAppOptions) {
+    const { host, port, database, user, password } = appOptions;
+
+    let url = host as String;
```

Comment:
```suggestion
    let url = String(host);
```
or
```suggestion
    let url = host.toString();
```
Are mildly safer, even though I don't expect host to be anything but a string.

Reply:
Right, the host is a string but the type is ambiguous so I'm only casting it instead of converting it to a string